content/08-mongodb/11-mongodb-transactions.mdx

```diff
@@ -22,25 +22,25 @@ If you are using [Prisma Client](https://www.prisma.io/docs/concepts/components/
 
 Transactions are a way to group together and isolate multiple statements for processing as a single operation. Instead of executing each command individually as it is sent to the server, in a transaction, commands are bundled together and executed in a separate context than other requests.
 
-Isolation is an important part of transactions. Within a transaction, the executed statements can only affect the environment within the transaction itself. From inside the transaction, statements can modify data and the results are immediately visible. From the outside, no changes are made until the transaction is committed, at which time all of the actions within the transaction become visible at once.
+[Isolation](/intro/database-glossary#isolation) is an important part of transactions. Within a transaction, the executed statements can only affect the environment within the transaction itself. From inside the transaction, statements can modify data and the results are immediately visible. From the outside, no changes are made until the transaction is committed, at which time all of the actions within the transaction become visible at once.
 
-These features help databases achieve [ACID compliance](https://www.prisma.io/dataguide/dataguide/datamodeling/in-vivo) by providing atomicity (actions in a transaction are either all committed or all rolled back) and isolation (outside of the transaction, nothing changes until the commit while inside, the statements have consequences). These together help the database maintain consistency (by guaranteeing that partial data transformations cannot occur). Furthermore, changes in transactions are not returned as successful until they are committed to non-volatile storage, which provides durability.
+These features help databases achieve [ACID compliance](https://www.prisma.io/dataguide/dataguide/datamodeling/in-vivo) by providing [atomicity](/intro/database-glossary#atomicity) and isolation (outside of the transaction, nothing changes until the commit while inside, the statements have consequences). These together help the database maintain [consistency](/intro/database-glossary#consistency) (by guaranteeing that partial data transformations cannot occur). Furthermore, changes in transactions are not returned as successful until they are committed to non-volatile storage, which provides [durability](/intro/database-glossary#durability).
```

Comment:
Not sure if this was intended, but there seems to be a bit of inconsistency here:

* For atomicity, you replaced the short explanation in parentheses with a glossary link
* For isolation, you left the short explanation, and didn't add a link
* For consistency, you added the link and kept the short explanation

Reply:
I thought I linked them all and removed the short explanations. That was my indentation. Will adjust